easybuild/easyconfigs/l/LAMMPS/LAMMPS-29Oct2020-fosscuda-2020a-Python-3.8.2-kokkos.eb

```
@@ -0,0 +1,107 @@
+name = 'LAMMPS'
+version = '29Oct2020'
+local_python_versionsuffix = '-Python-%(pyver)s'
+versionsuffix = local_python_versionsuffix
+versionsuffix += '-kokkos'
+
+homepage = 'https://lammps.sandia.gov/'
+description = """LAMMPS is a classical molecular dynamics code, and an acronym
+for Large-scale Atomic/Molecular Massively Parallel Simulator. LAMMPS has
+potentials for solid-state materials (metals, semiconductors) and soft matter
+(biomolecules, polymers) and coarse-grained or mesoscopic systems. It can be
+used to model atoms or, more generically, as a parallel particle simulator at
+the atomic, meso, or continuum scale. LAMMPS runs on single processors or in
+parallel using message-passing techniques and a spatial-decomposition of the
+simulation domain. The code is designed to be easy to modify or extend with new
+functionality.
+"""
+
+toolchain = {'name': 'fosscuda', 'version': '2020a'}
+toolchainopts = {'openmp': True, 'usempi': True, 'cstd': 'c++11'}
+
+# 'https://github.com/lammps/lammps/archive/'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = [
+    'stable_%(version)s.tar.gz',
+    {'filename': 'lammps_vs_yaff_test_single_point_energy.py', 'extract_cmd': "cp %s %(builddir)s"},
+]
+checksums = [
+    '759705e16c1fedd6aa6e07d028cc0c78d73c76b76736668420946a74050c3726',  # stable_29Oct2020.tar.gz
+    '5e5211f64113e8a38c102f8542fd59169b689fdb92afbff29c3b88f802f1a96c',  # lammps_vs_yaff_test_single_point_energy.py
+]
+
+local_source_dir_name = '%(namelower)s-%(version)s'
+
+builddependencies = [
+    ('CMake', '3.16.4'),
+    ('pkg-config', '0.29.2'),
+    ('archspec', '0.1.0', local_python_versionsuffix),
+]
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('libpng', '1.6.37'),
+    ('libjpeg-turbo', '2.0.4'),
+    ('netCDF', '4.7.4'),
+    ('GSL', '2.6'),
+    ('zlib', '1.2.11'),
+    ('gzip', '1.10'),
+    ('cURL', '7.69.1'),
+    ('HDF5', '1.10.6'),
+    ('tbb', '2020.1'),
+    ('PCRE', '8.44'),
+    ('libxml2', '2.9.10'),
+    ('FFmpeg', '4.2.2'),
+    ('Voro++', '0.4.6'),
+    ('kim-api', '2.1.3'),
+    ('Eigen', '3.3.7'),
+    ('yaff', {'arch=x86_64': '1.6.0', 'arch=POWER': False}, local_python_versionsuffix),
+    ('PLUMED', '2.6.0', local_python_versionsuffix),
+    ('ScaFaCoS', {'arch=x86_64': '1.0.1', 'arch=POWER': False}),
+    ('VTK', {'arch=x86_64': '8.2.0', 'arch=POWER': False}, local_python_versionsuffix),
```

```suggestion
    ('VTK', '8.2.0', local_python_versionsuffix),
```

I fixed VTK to build on POWER in #463, so we should be fine to use it as a dep here.